docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,692 @@
+        "published_at": "2021-04-07T10:10:31.949Z"
+      }
+    ]
+  },
+  {
+    "id": 8,
+    "name": "She's Cake",
+    "description": "She's Cake restaurant description in French",
+    "locale": "fr",
+    "localizations": []
+  }
+]
+```
+
+In the example response above:
+
+- restaurant with `"id": 4` is a French (`"locale": "fr"`) localization of the existing restaurant with `"id": 3` (created for the default `en` locale), as shown in the `localizations` object included in the response (see [creating a localization for an existing entry](#creating-a-localization-for-an-existing-entry)).
+- restaurant with `"id": "8"` was created from scratch using the API, passing the `locale: fr` in the request body (see [creating a new localized entry](#creating-a-new-localized-entry)).
+:::
+::::
+
+### Creating a new localized entry
+
+To create a localized entry from scratch, send a POST request to the Content API.
+
+If no locale has been passed in the request body, the entry is created using the default locale for the application:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Oplato",
+  "description": "Oplato restaurant description in English"
+}
+```
+
+:::
+
+::: tab Response
+
+**Example response**
+
+```json
+{
+    "id": 5,
+    "name": "Oplato",
+    "description": "Oplato restaurant description in English",
+    "locale": "en",
+    "localizations": []
+}
+```
+
+:::
+::::
+
+To create a localized entry for a locale different from the default one, add the `locale` attribute to the body of the POST request:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "She's Cake",
+  "description": "She's Cake restaurant description in French",
+  "locale": "fr"
+}
+```
+
+:::
+
+::: tab Response
+**Example response**
+
+```json
+{
+    "id": 8,
+    "name": "She's Cake",
+    "description": "She's Cake restaurant description in French",
+    "locale": "fr",
+    "localizations": []
+}
+```
+
+:::
+::::
+
+### Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
+|:----------------|:-----------------------------------------|
+| Collection type | `POST /{content-type}/:id/localizations` |  
+| Single type     | `POST /{content-type}/localizations`     |
+
+When creating a localization for existing localized entries, the body of the POST request can only accept localized fields.
+
+::: tip TIP
+The Content-Type should have the [`createlocalization` permission](http://localhost:8080/documentation/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md#collection-and-single-types) enabled, otherwise the POST request will return a `403: Forbidden` status.
```

Comment:
used a localhost:8080 address here :P

Reply:
Oh, nice catch! Thanks! [fixed it](https://github.com/strapi/documentation/pull/224/commits/14feba291ca9bab86b90e7df0e66d42d1e491159)